lib/src/box.dart

```diff
@@ -165,5 +166,55 @@ class Box<T> {
 
   QueryBuilder query(Condition qc) => QueryBuilder<T>(_store, _fbManager, _modelEntity.id.id, qc);
 
+  int count({int limit: 0}) {
+    Pointer<Uint64> _count = Pointer<Uint64>.allocate();
+    try {
+      checkObx(bindings.obx_box_count(_cBox, limit, _count));
+      return _count.load<int>();
+    } finally {
+      _count.free();
+    }
+  }
+
+  bool isEmpty() {
+    Pointer<Uint8> _isEmpty = Pointer<Uint8>.allocate();
+    try {
+      checkObx(bindings.obx_box_is_empty(_cBox, _isEmpty));
+      return _isEmpty.load<int>() > 0 ? true : false;
+    } finally {
+      _isEmpty.free();
+    }
+  }
+
+  bool remove(int id) {
+    try {
+      checkObx(bindings.obx_box_remove(_cBox, id));
+    } on ObjectBoxException catch (ex) {
```

Comment:
There's nothing exceptional about removing an ID that's already removed so that's why we're returning `bool false` in that case. I've changed the code though, there was no need for exception catching after all.